images/metering-ansible-operator/roles/meteringconfig/tasks/configure_tls.yml

```
@@ -25,3 +25,23 @@
     file:
       path: "{{ certificates_dir.path }}"
       state: absent
+
+- name: Log Events for validating TLS
+  k8s_event:
+    state: present
+    name: test-https-emily109
+    namespace: {{ meta.namespace }}
```

You need quotes around this variable:
```suggestion
    namespace: "{{ meta.namespace }}"
```